persistence-api/src/main/java/io/stargate/db/query/builder/ValueModifier.java

```diff
@@ -32,9 +32,18 @@ public static ValueModifier of(String columnName, Value<?> value) {
         .build();
   }
 
+  public static ValueModifier of(String columnName, Value<?> value, Operation operation) {
```

Comment:
Postfix notation :). It made sense to me to put it at the end since the method above is `public static ValueModifier of(String columnName, Value<?> value) {...}`